app/models/health/qualifying_activity_v2.rb

```diff
@@ -217,5 +217,15 @@ def internal_compute_valid_unpayable
 
       reasons.uniq
     end
+
+    def place_of_service
+      return '11' if telehealth?
```

Comment:
Does it matter that 11 is a string here, but an integer in app/models/health/qualifying_activity_base.rb ?